recipes/SimpleElastix/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.10.0" %}
+
+package:
+  name: simpleelastix
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/kaspermarstal/SimpleElastix.git
+  git_tag: 18058e48a021d4bc79924a0c6a3826a85f5dfab9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - python
+    - cmake    >=3.0  # [not win]
+    - libtiff  4.0.*  # [not win]
+    - libpng   1.6.*  # [not win]
+    - jpeg     8*  # [not win]
+    - zlib     1.*  # [not win]
+    - setuptools
+  run:
+    - python
+    - libtiff  4.0.*  # [not win]
+    - libpng   1.6.*  # [not win]
+    - jpeg     8*  # [not win]
+    - zlib     1.*  # [not win]
```

Comment:
Please update the pinnings using this [listing]( https://github.com/conda-forge/conda-forge.github.io/blob/49a67ac9b798d6d587207a28f65fa81fe6a3ce57/scripts/pin_the_slow_way.py#L32-L73 ).